README.md

```diff
@@ -148,3 +149,29 @@ unchanged from [/v2/links/view](#v2linksview)
 ## /v3/links/click
 
 unchanged from [/v2/links/click](#v2linksclick)
+
+## /v3/links/activity-stream
+
+The `activity-stream` endpoint captures any event takes place in the Activity Stream Addon.
+
+Method: POST
+
+Example Payload:
+
+    {
+      "client_id": "some_client_id",
+      "addon_version": "1.0",
+      "tab_id": 1,
+      "load_reason": "newtab",
+      "source": "activity_stream",
+      "search": 0,
+      "top_site_click": 1,
```

Comment:
this doesn't need to exist, and neither does `spotlight_click`, `recent_item_click`